spec/rails_helper.rb

```diff
@@ -0,0 +1,63 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
```

Comment:
- [ ] I am adding this here, there is no spec file for likes and comments, kindly add them